plugins/DataStreamCereal/cereal_parser.hpp

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <PlotJuggler/messageparser_base.h>
+#include <capnp/dynamic.h>
+#include <capnp/schema.h>
+#include <capnp/serialize.h>
+#include <iostream>
+
+#ifndef DYNAMIC_CAPNP
+#define DYNAMIC_CAPNP  // Do not depend on generated log.capnp.h structure
+#endif
+#include "common.h"
```

Comment:
where do these come from?

Reply:
They were added in commit e19cf5d35e80bb8f604d764e6d5019e101b3699e, going to remove this anyway, it's just an unused copy of rlog_parser.